plugins/org.eclipse.elk.core.debug/src/org/eclipse/elk/core/debug/views/graph/LayoutGraphViewDropAdapter.java

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Kiel University and others.
```

Comment:
Change "Kiel University" to your employer.

Reply:
done